pkg/config/config.go

```diff
@@ -123,6 +124,23 @@ func (BucketType) ID() TypeId {
 	return BucketTypeId
 }
 
+type DocumentTypeType string
+
+const (
+	DashboardType DocumentTypeType = "dashboard"
+	NotebookType  DocumentTypeType = "notebook"
+)
+
+type DocumentType struct {
```

Comment:
This could just be:

`type DocumentType string `

and `DocumentTypeType` can be removed.